src/components/ChatLog.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import ChatEntry from './ChatEntry';
+import './ChatLog.css';
+
+
+
+const ChatLog = (props) => {
+   
+    const getMessage = props.entries.map((message) =>{
+        
+        return (
+            <ChatEntry id={message.id} sender = {message.sender} key = {message.id}
+            body = {message.body} timeStamp = {message.timeStamp } liked ={message.liked} onLiked = {props.likedFunc} />
```

Comment:
2 notes: Whitespacing between equal signs are inconsistent and need to be cleaned up. Also, the line spacing here is a little hard to read.

To make this more readable, each prop should be on its own line, like:
```
<ChatEntry 
    id={message.id} 
    sender={message.sender} 
    key={message.id}
    body={message.body} 
    timeStamp={message.timeStamp } 
    liked={message.liked} 
    onLiked={props.likedFunc} 
/>